### PR TITLE
Address form input

### LINKED ISSUE #43: Search Location feature to Create Marker form
Adapt the search box that is attached to the map to allow the user to search for where they want to place the marker (inside the form)

### PR DESCRIPTION
**Pull request by TBarrett553**
Updated the marker creation form to to allow the user to search for the location of the marker.

### CODE REVIEW

File: maptivist/src/main/java/com/google/sps/Marker.java [hunk 1]
```diff
@@ -223,70 +238,70 @@ public Entity toEntity() {
...
 
-    public String createCategoriesString(Set<String> categories){
-        String categoryString = "";
-        Base64.Encoder encoder = Base64.getEncoder();  
-        for (String category : categories) {
-            String categoryByte = encoder.encodeToString(category.getBytes());  
-            categoryString += categoryByte + ",";
-        }
-        return categoryString.substring(0, categoryString.length() - 1);     
+  public String createCategoriesString(Set<String> categories){
```
Comment (flying-bus) [inline maptivist/src/main/java/com/google/sps/Marker.java:241]: There are a lot of formatting-only changes in this file.

General advice for the future:
A good practice is to make formatting changes in a separate PR from code changes. For the formatting PR, the reviewers can just give a quick skim, and then focus only on the code changes in the second PR.

**Review by flying-bus — APPROVED** [review]
Great job!

Doesn't having the toEntity and Marker constructor from Entity in the class make adding fields easier? :)

**Review by ayomide321 — APPROVED** [review]

File: maptivist/src/main/java/com/google/sps/Marker.java (hunk 1 above)
Comment (TBarrett553) [reply maptivist/src/main/java/com/google/sps/Marker.java:241]: Ok cool that makes sense! When you mean formatting do you mean the first commits where I just rearranged the search box? 

Also, yes the update to the Marker Class is really great! Thank you for that idea!

Comment (flying-bus) [reply maptivist/src/main/java/com/google/sps/Marker.java:241]: By formatting, I just mean the whitespace/indentation around the code. In this section, it looks like the only change was the indentation.

**Pull request merged**